packages/kununu-utils/middlewares/expressLogger/expressLogger.spec.js

```diff
@@ -1,36 +1,80 @@
+import {advanceTo, clear} from 'jest-date-mock';
+
 import expressLogger from './index';
 
 const express = require('express');
 const request = require('supertest');
 
+process.env.NODE_ENV = 'production';
+
 describe('Express logger', () => {
   const app = express();
-  let originalEnv;
 
-  app.get('/', expressLogger('app-reviews'), (req, res) => {
+  app.get('/', expressLogger('app-example'), (req, res) => {
     res.send();
   });
 
+  const spyFunc = jest.fn();
+
+  global.console = {log: spyFunc};
+
   beforeAll(() => {
-    originalEnv = process.env.NODE_ENV;
-    process.env.NODE_ENV = 'production';
+    advanceTo(new Date(2019, 1, 1, 0, 0, 0));
   });
 
   afterAll(() => {
-    process.env.NODE_ENV = originalEnv;
+    clear();
+  });
+
+  afterEach(() => {
+    spyFunc.mockClear();
   });
 
-  it('Logs a request with the expected info', async () => {
-    const spyFunc = jest.fn();
+  it('logs a request in', async () => {
```

Comment:
I think it makes sense to extend this test to see when a request comes in and out, since they have different structure. e.g.: different message, request in has no `res`...